hack/testing/test-020-olm-upgrade.sh

```diff
@@ -26,13 +26,14 @@ TIMEOUT_MIN=$((2 * $minute))
 NAMESPACE="openshift-logging"
 manifest_dir=${repo_dir}/manifests
 version=$(basename $(find $manifest_dir -type d | sort -r | head -n 1))
+previous_version=$(echo $version | awk '{print ($1*10-1)/10.0}')
```

Comment:
not sure why you need to have the extra math in there - assuming `$version` is `N.M` where `N` and `M` are digits:
```
echo "" | awk -v ver=4.2 '{print ver + 0.1}'
4.3
```

Reply:
right